theano/gof/cc.py

```diff
@@ -1183,8 +1169,8 @@ def make_thunk(self, input_storage=None, output_storage=None,
             be allocated.
         storage_map: dict that map variables to storages.
             This is used when you need to customize the storage of
-            this thunk.
-
+            this thunk.keep_lock
+        WRITEME
```

Comment:
Bad indentation. Here is what you should write while at it:

keep_lock
    If True, we won't release the lock on the compiledir at the end of this function call.